dash/dash-renderer/src/libraries/LibraryManager.tsx

```diff
@@ -44,14 +45,18 @@ const LibraryManager = (props: LibrariesManagerProps) => {
     useEffect(() => {
         if (layout && !isEmpty(layout) && !ready && !initialLibraries) {
             const libraries: string[] = [];
-            crawlLayout(layout, (child: any) => {
+            const addLib = (child: any) => {
                 if (child.namespace && !libraries.includes(child.namespace)) {
                     libraries.push(child.namespace);
                 }
-            });
+            };
+            crawlLayout(layout, addLib);
+            if (validation_layout) {
+                crawlLayout(validation_layout, addLib);
```

Comment:
Doesn't this defeat the purpose though? The difference between `validation_layout` and `layout` is just items that we're not going to show on the initial view.